dsgrid/registry/dimension_registry_manager.py

```diff
@@ -237,9 +248,8 @@ def _register(self, config, submitter, log_message, context):
             registration.version,
         )
 
-        context.add_ids(RegistryType.DIMENSION, dimension_ids, self)
-        dimension_ids.extend(existing_ids)
```

Comment:
This was the bug. The function was written to not care about order but the project registry manager expected them to be in order.